render-site-map.R

```diff
@@ -54,7 +54,10 @@ render_site_map <- function(traits, render_date, legend_title, overlay_image = 0
         scan_number <- which(full_image_paths == path)
         scan_name <- paste0('scan ', scan_number)
         fullfield_image <- brick(path)
-        map <- viewRGB(x = fullfield_image, map = map, layer.name = scan_name)
+        map <- viewRGB(x = fullfield_image, 
+                       map = map, 
+                       quantiles = NULL, 
```

Comment:
also need to add 

```r
r = 1, g = 2, b = 3
```
to get original color of thumb

Reply:
done